npm/cypress-schematic/src/schematics/ng-add/files-core/cypress.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from 'cypress'
```

Comment:
Can we suffix all template files with `.template` or `__template__` (not sure which), then we wouldn't have to worry about eslint ignores or tsconfig excludes

Reply:
Yes. I added `.template` and then updated the eslint ignore file.